.changeset/tall-eagles-heal.md

```diff
@@ -0,0 +1,10 @@
+---
+"@guardian/cdk": patch
+---
+
+Bump dependencies"
+  - e905b31: bump typedoc from  0.26.7 to 0.26.8
+  - a334ea9: bump @types/node from 22.7.4 to 22.7.5
+  - 504fe94: bump @changesets/cli from 2.27.8 to 2.27.9
+  - 5a5498d: bump @types/node from 22.7.0 to 22.7.4
+  - 7dbd8ac: bump @types/node from 22.5.5 to 22.7.0
```

Comment:
These are all dev dependencies; I'm surprised to see them causing a version change.

Reply:
Ah yea, good point! Removed!